Base/usr/share/man/man1/bt.md

```diff
@@ -21,6 +21,22 @@ addresses for each frame in the stack producing a backtrace.
 * If Kernel addresses are available, they will not be symbolicated unless
   the current user has access to the `/boot/Kernel` file.
 
+## See also
+
+* [`Inspector`(1)](Inspector.md)
+
+* [`Profiler`(1)](Profiler.md)
+
+* [`watch`(1)](watch.md)
+
+## Arguments
+
+* `pid`: PID of process
```

Comment:
I'd rather use something like: "id of process" / "process id". PID of process reads to me like: process id of process.

Reply:
Done.